x-pack/plugins/files/common/types.ts

```diff
@@ -137,6 +135,25 @@ export type BaseFileMetadata = {
   Compression?: FileCompression;
 };
 
+/**
+ * Metadata add by the files service.
+ */
+export interface FileSystemMetadata {
+  /**
+   * The human-friendly user name of the owner of the file
+   *
+   * @note this field cannot be used to uniquely ID a user. See {@link FileSystemMetadata['uid']}.
+   */
+  owner?: string;
```

Comment:
if we keep `username`, I'd suggest we be consistent in field naming.
I'd call it `username` as it is in es, as `uid` is exactly how it is there. Or maybe better grouped under `creator` or `author` 
(nit: I think it better reflects what it does then the `owner`)

```
meta: { creator: {
 username: string,
 uid: string:
}}
```

Reply:
I got `owner` from the ECS specification for [field names for files](https://www.elastic.co/guide/en/ecs/current/ecs-file.html).

By placing the `owner` field inside of `meta` we are already breaking a bit from their pattern (it should be `file.owner`, not `file.meta.owner`. Perhaps it is worth considering doing something different in this case, however I think if it were not for the SO field limit issue we probably should have placed `owner` at the the same level as other "system" fields, not inside meta.

In the case of `uid` I think this is fairly safe to keep the way it is since the definition is in the name uid = "user ID".

Reply:
All to say: I think we should stick with `owner` for now 😄

Reply:
Ah, didn't know about https://www.elastic.co/guide/en/ecs/current/ecs-file.html. 
I guess sound good then